SparkPost/2web2ui
# EO-764 Adjust signals subtitles, Remove other charts header

### What Changed
- Removed title from OtherChartsHeader, renamed to Divider
- Adjusted page subtitles to be more clear

### How To Test
- [Point your local upstreams to staging or production](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or account 108 for data
- Visit all signals details pages
- Verify page subtitles with different facets, and with/without a subaccount

### To Do
